test/unittests/execute_call_test.cpp

```diff
@@ -171,7 +171,7 @@ TEST(execute_call, call_indirect_imported_table)
         "0061736d01000000010a026000017f60017f017f020a01016d01740170010514030201010a0901070020001100"
         "000b");
 
-    const Module module = parse(bin);
+    const auto module = parse(bin);
```

Comment:
Can these thing be really `const` when it is a `unique_ptr`?

Reply:
No, because you cannot move a const object.

Reply:
Yeah, const unique_ptr means you can't change the wrapped pointer.